README.md

```diff
@@ -1,4 +1,19 @@
-# Python boilerplate for GitHub tasks
+# Images and custom auth token serializer
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+- Download [ModHeader](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj?hl=en)
+
+### In this task you will work with images and add auth token serializer
+
+1. Add ImageField `image` to the `Movie` model.
+   - Upload images should be only available with `/upload-image/` endpoint.
+Image field should not be available on POST `api/cinema/movies/`.
+   - Image url should be shown on:
+     - Movie list and detail pages
+     - Movie session detail page
+2. Replace username field with email field in User model. Copy tests from
+`token-authentication` task but replace username with email where it is needed.
```

Comment:
I think, that you'd better do it. It will be more logical for them just to run tests to check if everything is working